### PR TITLE
quicker quick start + request to add to core contributors

### PR DESCRIPTION
**Pull request by davidmarkclements**
I stumbled on an `npm init` feature, if you do `npm init <name>` npm will execute the bin file of `create-<name>`.

Created http://npm.im/create-fastify to hook into that functionality, it just uses the generate cli function of fastify-cli.

So now if you do `npm init fastify` in a folder, it'll autogenerate a fastify project.

Before this can land we need to move https://github.com/davidmarkclements/create-fastify to https://github.com/fastify/create-fastify. 

As a form of request, I've also added myself to the list of core contributors.

Here's my case for making this request:

* fastify is literally the web framework of my dreams.
* I'm familiar with the internals
* I have a corpus of performance-related OSS and published analysis
* pino is a core part of fastify and I think we can do more to integrate
* avvio contribution (https://github.com/mcollina/avvio/pull/92)
* the fastify community has an amazing vibe, and I love to collaborate with value-centric and considerate people (https://twitter.com/matteocollina/status/1221795471431217153)

If this is acceptable, I'd like to perform a full documentation audit and rewrite (I have this provisionally scheduled for early-April). I have authoring/technical writing experience that can help here.

#### Checklist

- ~~run `npm run test` and `npm run benchmark`~~
- ~~tests and/or benchmarks are included~~
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by gyszalai** [comment]
@davidmarkclements the link http://npm.im/fastify-create is wrong, I guess it's http://npm.im/create-fastify

**Comment by davidmarkclements** [comment]
@gyszalai oops! edited to update :+1: thank you

File: README.md [hunk 1]
```diff
@@ -199,6 +215,7 @@ Team members are listed in alphabetical order.
 * [__Cemre Mengu__](https://github.com/cemremengu), <https://twitter.com/cemremengu>, <https://www.npmjs.com/~cemremengu>
 * [__Manuel Spigolon__](https://github.com/eomm), <https://twitter.com/manueomm>, <https://www.npmjs.com/~eomm>
 * [__James Sumners__](https://github.com/jsumners), <https://twitter.com/jsumners79>, <https://www.npmjs.com/~jsumners>
+* [__David Mark Clements__](https://github.com/davidmarkclements), <https://twitter.com/davidmarkclem>, <https://www.npmjs.com/~davidmarkclements>
```
Comment (mcollina) [inline README.md]: Names ordered by surname

Comment (jsumners) [reply README.md]: 🤦‍♂

Comment (davidmarkclements) [reply README.md]: oh! lol

**Comment by davidmarkclements** [comment]
- 2fa npm/gh already enabled
- added to website https://github.com/fastify/website/pull/156
- placed name/detials as per alphabetical ordering (force push)
- requested membership of fastify team
- create-fastify repo transferred https://github.com/fastify/create-fastify

**Pull request merged**